src/md/bylaws.md

```diff
@@ -0,0 +1,381 @@
+For PMC decisions, only the votes of PMC members are binding.
+
+Voting can also be applied to changes made to the Apex codebase. These typically
+take the form of a veto (-1) in reply to the commit message sent when the commit is
+made.
+
+###Approvals
+
+These are the types of approvals that can be sought. Different actions require
+different types of approvals:
+
+<table border="0" class="table table-striped">
+<tbody>
+  <tr class="a">
+    <td>Consensus</td>
+
+    <td>For this to pass, all voters with binding votes must vote and there can
+    be no binding vetoes (-1). Consensus votes are rarely required due to the
+    impracticality of getting all eligible voters to cast a vote.</td>
+  </tr>
+
+  <tr class="b">
+    <td>Lazy Consensus</td>
+
+    <td>Lazy consensus requires 3 binding +1 votes and no binding vetoes.</td>
+  </tr>
+
+  <tr class="a">
+    <td>Lazy Majority</td>
+
+    <td>A lazy majority vote requires 3 binding +1 votes and more binding +1
+    votes that -1 votes.</td>
+  </tr>
+
+  <tr class="b">
+    <td>Lazy Approval</td>
+
+    <td>An action with lazy approval requires at least 1 binding +1 vote unless a
+    -1 vote is received, at which time, depending on the type of action, either
+    lazy majority or lazy consensus approval must be obtained.</td>
+  </tr>
+
+  <tr class="a">
+    <td>2/3 Majority</td>
+
+    <td>Some actions require a 2/3 majority of active committers or PMC members
+    to pass. Such actions typically affect the foundation of the project (e.g.
+    adopting a new codebase to replace an existing product). The higher threshold
+    is designed to ensure such changes are strongly supported. To pass this vote
+    requires at least 2/3 of binding vote holders to vote +1</td>
+  </tr>
+</tbody>
+</table>
+
+###Vetoes
+
+A valid, binding veto cannot be overruled. If a veto is cast, it must be
+accompanied by a valid reason explaining the reasons for the veto. The validity of
+a veto, if challenged, can be confirmed by anyone who has a binding vote. This does
+not necessarily signify agreement with the veto - merely that the veto is
+valid.
+
+If you disagree with a valid veto, you must lobby the person casting the veto to
+withdraw their veto. If a veto is not withdrawn, the action that has been vetoed
+must be reversed in a timely manner.
+
+###Actions
+
+This section describes the various actions which are undertaken within the
+project, the corresponding approval required for that action and those who have
+binding votes over the action.
+
+<table border="0" class="table table-striped">
+<thead>
+  <tr class="a">
+    <th>Action</th>
+
+    <th>Description</th>
+
+    <th>Approval</th>
+
+    <th>Binding Votes</th>
+  </tr>
+</thead>
+
+<tbody>
+  <tr class="b">
+    <td>Code Change</td>
+
+    <td>A change made to a codebase of the project and committed by a committer.
+    This includes source code, documentation, website content, etc.</td>
+
+    <td>Lazy approval and then Lazy consensus.</td>
```

Comment:
Shoddy just be "Lazy consensus" I think but in most case consensus is not really required except for large changes. Any commit can be easily reverted if it causes an issues.